.rubocop.yml

```diff
@@ -41,6 +41,8 @@ Jekyll/NoPutsAllowed:
 # ----------------------------------------
 Gemspec/DeprecatedAttributeAssignment:
   Enabled: true
+Gemspec/DevelopmentDependencies:
+  Enabled: false
```

Comment:
I wonder if this could be a good cleanup in the future? Not sure the value of add_development_dependency anymore in a gemspec.

Reply:
I was always in favor of moving dev-deps to Gemfile until I saw your [`commit @ rubocop-jekyll`](https://github.com/jekyll/rubocop-jekyll/pull/27/commits/34338391b2aa35d2c09a5b71d8b77bb3115d3196)

Reply:
I see -- I think it was more that I was too lazy to move them all to the Gemfile. I think managing dev dependencies in the Gemfile in 2024 makes sense, but no need to enforce it now if it's going to be a good amount of work (especially across repos).